pytest.ini

```diff
@@ -2,4 +2,5 @@
 markers =
   aws
   infra
+  config_value
```

Comment:
Dont think this needs to be here.  This is the one for unit tests not validations.  It doesnt need to be in the validations.ini either because its registered here:

https://github.com/cloudera-labs/cdpctl/blob/1d763ba2ea431dbff396ddc32fd46d025f57693c/cdpctl/validation/conftest.py#L206-L210

Reply:
We might just be able to remove the whole file for now

Reply:
> Dont think this needs to be here. This is the one for unit tests not validations. It doesnt need to be in the validations.ini either because its registered here:
> 
> https://github.com/cloudera-labs/cdpctl/blob/1d763ba2ea431dbff396ddc32fd46d025f57693c/cdpctl/validation/conftest.py#L206-L210

I added that because I was seeing these when I ran `pytest tests`:

```
cdpctl/validation/infra/validate_aws_security_groups.py:138
  /workspaces/cdpctl/cdpctl/validation/infra/validate_aws_security_groups.py:138: PytestUnknownMarkWarning: Unknown pytest.mark.config_value - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.config_value(path="env:tunnel", value=False)
```

Reply:
Ahh it looks like the unit tests are finding the "tests" in the validation paths.  we probably want to limit that.

Reply:
> We might just be able to remove this whole markers section for now

we get million warnings for markers for `pytest tests` when we remove the markers section. There is also a `pytests.ini` btw, not sure why we have both of these files.

Reply:
despite this? `norecursedirs = cdpctl/validation/*`

Reply:
Im guessing its erroring out when it finds them from an include actually.

I only think we need the pytest.ini.  We can remove the other.

Reply:
removed pytests.ini